dyson/solvers/chempot.py

```diff
@@ -120,6 +120,56 @@ def get_greens_function(self):
         return self.gf.copy(chempot=self.chempot, deep=False)
 
 
+class AufbauPrincipleBisect(AufbauPrinciple):
```

Comment:
copy the doc string for aesthetics (i know it inherits `__doc__` but nicer to be able to read it in the source code). Maybe add another line saying this uses bisection